pyccel/commands/pyccel_wrap.py

```diff
@@ -13,50 +13,30 @@
 import argparse
 import pathlib
 
-__all__ = ['pyccel_wrap_command']
+from .argparse_helpers import add_version_flag, add_accelerator_selection, add_compiler_selection
+from .argparse_helpers import check_file_type, add_common_settings
 
-#==============================================================================
-def pyccel_wrap_command() -> None:
-    """
-    Pyccel console command.
+__all__ = ['pyccel_wrap_command']
 
-    The Pyccel console command allows translating Python files using Pyccel in
-    a command-line environment. This function takes no parameters and sets up
-    an argument parser for the Pyccel command line interface.
+PYCCEL_WRAP_DESCR = 'Create the wrapper to allow code to be called from Python.'
 
-    The command line interface requires a Python file to be specified, and it
-    supports various options such as specifying the output language (C,
-    Fortran, or Python), compiler settings, and flags for accelerators like
-    MPI, OpenMP, and OpenACC. It also includes options for verbosity,
-    debugging, and exporting compile information. Unless the user requires the
-    process to stop after a specific stage, Pyccel will execute the full
-    translation and compilation process until a C Python extension module is
-    generated, which can then be imported in Python. In addition, if the input
-    file contains an `if __name__ == '__main__':` block, an executable will be
-    generated for the corresponding block of code.
+def setup_pyccel_wrap_parser(parser):
     """
+    Add the pyccel-wrap arguments to the parser.
 
-    parser = argparse.ArgumentParser(description="Pyccel's command line interface.",
-                      add_help=False)
+    Add the pyccel-wrap arguments to the parser for command line arguments.
```

Comment:
This function is not only used by the deprecated `pyccel-wrap` command, but also by the new `pyccel wrap` subcommand. Hence I would update the docstring

```suggestion
def setup_pyccel_wrap_parser(parser):
"""
    Add the `pyccel wrap` arguments to the parser.

    Add the `pyccel wrap` arguments to the parser for command line arguments.
```